docs/Tutorial_Top_Ten_3.ipynb

```diff
@@ -0,0 +1,251 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #3.    idx_motifA = np.argmin(mp[:, 0])
</pre><p>I think we can go with simple name <code>idx</code> but then add comment: </p><pre class="ql-syntax" spellcheck="false">idx = np.argmin(mp[:, 0])  # idx : start index of motif
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/TDAmeritrade/stumpy/pull/676/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>